recipes/hedley/all/test_package/example.cpp

```diff
@@ -0,0 +1,5 @@
+#include "hedley.h"
+
+int main() {
+    return !(HEDLEY_VERSION==15);
+}
```

Comment:
```suggestion
#include <iostream>

int main() {
    std::cout << "HEDLEY_VERSION: " << HEDLEY_VERSION << std::endl;
    return 0;
}

```

Don't force a single version. For any different version it won't work.

Reply:
Is there a way to check it matches the version that conan thinks it's installing?

Reply:
We do not need to, there would only be one source that can be packed.

Conan is well tested for we rely on testing the package (header being available in this case)

Reply:
I hope to add other versions in the `conandata.yml` file. Would it make sense to test the versions match after that?

Reply:
Nope! Conan downloads each source into separate folders and takes care of all that for you